### PR TITLE
Implement active/inactive whitelist toggle

### LINKED ISSUE #3181: Implement active/inactive whitelist logic
## Specification
* Add toggle button (With popover) to Manage whitelist modal
* Add logic of switching between active/inactive
* Show proper warning for inactive state

### Design (optional, might not be applicable)
https://www.figma.com/file/Phby7kEYihhvGgQn69A314/Verified-Recipients?node-id=96%3A13720
https://www.figma.com/file/Phby7kEYihhvGgQn69A314/Verified-Recipients?node-id=81%3A24210

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
Added active/inactive toggle to `ManageWhitelistDialog`

FireShot Capture 540 - Colony - localhost

FireShot Capture 541 - Colony - localhost

Resolves #3181

### CODE REVIEW

**Review by arrenv — CHANGES_REQUESTED** [review]
@ArmandoGraterol looks good, the Tooltip looks a little different though. I'm pretty sure sizing and weight was just pulled from the spec, however, everywhere else in the DApp it is consistent. So, probably better to match the DApp.

See screenshots below:

aaeb4e59-32f4-41f3-810a-0d51eb2f2d82

7a1d8270-c123-4dbd-9164-676a82ce9b54

**Review by danbr — CHANGES_REQUESTED** [review]
Looks good - one small detail to check. The positioning of the `question-mark` icon seems too high and not centered.

Screenshot 2022-02-20 at 12 50 14

Vs Figma

Screenshot 2022-02-20 at 12 50 52

**Comment by ArmandoGraterol** [comment]
@danbr For me, it's not looking like that:

FireShot Capture 543 - Colony - localhost

What OS are you using?

**Comment by danbr** [comment]
> @danbr For me, it's not looking like that:
> 
> FireShot Capture 543 - Colony - localhost
> 
> What OS are you using?

@ArmandoGraterol Oh no - thats annoying. I'm running MacOS. Isnt there a similar issue #2825. 
Not sure how you wish to proceed, but i can approve and then when #2825 is solved we can try applying the fix here (or vice-versa)

**Review by danbr — APPROVED** [review]

File: src/modules/dashboard/components/Dialogs/CreatePaymentDialog/CreatePaymentDialogForm.tsx
Comment (dit7ya) [inline src/modules/dashboard/components/Dialogs/CreatePaymentDialog/CreatePaymentDialogForm.tsx:410]: Quick nitpicky question, should this be part of this PR?

**Review by dit7ya — APPROVED** [review]
LGTM. I am guessing this PR is just for the UI and not the business logic.

Also, on aligning things vertically, I think we should have a simple abstracted away solution that works cross-system without much hassle. What do you guys think?

**Review by chinins — COMMENTED** [review]
Nice work but I have the same problem as Dan - the alignment of the question mark tooltip. It's the same old problem and the looks depends on the OS system. I thought that it was solved somewhere else, wasn't it?
Screenshot 2022-02-22 at 16 12 04

File: src/modules/dashboard/components/Dialogs/CreatePaymentDialog/CreatePaymentDialogForm.tsx
Comment (ArmandoGraterol) [reply src/modules/dashboard/components/Dialogs/CreatePaymentDialog/CreatePaymentDialogForm.tsx:410]: It's not included in the issue no, this is a fix for an unrelated bug I found while working on the issue

**Comment by ArmandoGraterol** [comment]
> Nice work but I have the same problem as Dan - the alignment of the question mark tooltip. It's the same old problem and the looks depends on the OS system. I thought that it was solved somewhere else, wasn't it? Screenshot 2022-02-22 at 16 12 04

Most likely yes! I'm gonna take a look at the other PRs to see what's up

File: src/modules/dashboard/components/Dialogs/ManageWhitelistDialog/ManageWhitelistActiveToggle.tsx
Comment (alicjakujawa) [inline src/modules/dashboard/components/Dialogs/ManageWhitelistDialog/ManageWhitelistActiveToggle.tsx:34]: What actually this name means?  maybe something a bit more clear like whitelistActivated or isWhiletlistActivated?

File: src/modules/dashboard/components/Dialogs/ManageWhitelistDialog/ManageWhitelistActiveToggle.tsx
Comment (alicjakujawa) [inline src/modules/dashboard/components/Dialogs/ManageWhitelistDialog/ManageWhitelistActiveToggle.tsx:47]: Same here whitelistStatus with values true or false, it doesn't tell too much. Again maybe something like whitelistActivated (true/false)

**Review by alicjakujawa — APPROVED** [review]
Small comments, after that its fine to merge

**Review by arrenv — APPROVED** [review]
@ArmandoGraterol I have the alignment issues as well, however, I think the `       QuestionMarkTooltip` in `Toggle.tsx` can be looked at in the existing https://github.com/JoinColony/colonyDapp/issues/2825 issue, which you seemed to have assigned yourself to recently.

**Pull request merged**
